.github/workflows/int-and-connected-test-run.yml

```diff
@@ -30,8 +36,46 @@ on:
   schedule:
     - cron: '0 4 * * *' # run at 4 AM UTC, 12PM EST.
 jobs:
+  test_check:
+    name: "Checkout, verify and run unit tests"
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+    if: "!contains( github.event.pull_request.labels.*.name, 'skip-ci')"
+    runs-on: ${{ matrix.os }}
+    ## skips if pr label is 'skip-ci'
+    # run a local Postgres container in Docker for the basic check tests
+    services:
+      postgres:
+        image: postgres:11
```

Comment:
Does the image: postgres:11 imply we're using postgres 11 to test with?  Does it matter that the [setup instructions](https://github.com/DataBiosphere/jade-data-repo/blob/develop/docs/jade-getting-started.md#10-install-postgres-12) indicate postgres v 12.

Reply:
Hmm...good catch.  Let me try to bump that to 12

Reply:
Actually, the documentation is wrong.  We use 11 to run in prod